vitest.config.mjs

```diff
@@ -41,7 +41,14 @@ export default defineConfig({
                 lines: 95,
                 statements: 95,
             },
-            reporter: ['clover', 'json', 'lcov', 'text', ['text', { file: 'coverage.txt' }]],
+            reporter: [
+                'clover',
+                'html',
+                'json',
+                'lcov',
+                'text',
+                ['text', { file: 'coverage.txt' }],
```

Comment:
Not necessarily for this PR, but this is a hack for not having a markdown reporter for jest. But there is [one for vitest](https://github.com/pecirep/vitest-markdown-reporter), so we might switch to that, eventually. (Or not -- has anyone ever looked at the coverage report in CI since we added that? 😅)

Reply:
Good to know! And yeah, markdown would be better... or a GitHub Action or something. I'd also love to figure out how to get our coverage reports from Vitest and Karma merged.